docs/install_guides/installing-openvino-pip.md

```diff
@@ -1,13 +1,39 @@
 # Install Intel® Distribution of OpenVINO™ Toolkit from PyPI Repository {#openvino_docs_install_guides_installing_openvino_pip}
 
-OpenVINO™ toolkit is a comprehensive toolkit for quickly developing applications and solutions that solve a variety of tasks including emulation of human vision, automatic speech recognition, natural language processing, recommendation systems, and many others. Based on the latest generations of artificial neural networks, including Convolutional Neural Networks (CNNs), recurrent and attention-based networks, the toolkit extends computer vision and non-vision workloads across Intel® hardware, maximizing performance. It accelerates applications with high-performance AI and deep learning inference deployed from edge to cloud.
+You can install Intel® Distribution of OpenVINO™ Toolkit through the PyPI repository, including both OpenVINO Runtime and OpenVINO Model Development Tools. Besides, from the 2022.1 release, OpenVINO Model Development Tools can only be installed via PyPI.
 
-Intel® Distribution of OpenVINO™ Toolkit provides the following packages available for installation through the PyPI repository:
 
-* Runtime package with the Inference Engine inside: [https://pypi.org/project/openvino/](https://pypi.org/project/openvino/)
-* Developer package that includes the runtime package as a dependency, Model Optimizer, Accuracy Checker and Post-Training Optimization Tool: [https://pypi.org/project/openvino-dev](https://pypi.org/project/openvino-dev)
+## Installing OpenVINO Runtime
 
-## Additional Resources
+The OpenVINO Runtime package contains the Inference Engine. Use the following command:
+```
+pip install openvino
+```
+
+For system requirements and more detailed steps, see <https://pypi.org/project/openvino>.
+
+
+## Installing OpenVINO Model Development Tools
```

Comment:
It would be good to clarify that installing openvino-dev also installs the runtime package openvino. openvino is a dependency of openvino-dev. So users do not have to separately install the runtime package.

Reply:
Agree. Adding a sentence to state that. :)

Reply:
Aha, I checked elsewhere and got the information "To install OpenVINO Model Development Tools, you must have OpenVINO Runtime installed as a prerequisite." @slyubimt Hi Sergey, could you help clarify whether Runtime will be automatically installed or the user should install it first?